test/infra/aws/export.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o pipefail
+set -o nounset
+
+export AWS_VPC_ID="$("${TERRAFORM_BIN}" -chdir=.local/infra/aws output -raw vpc_id)"
+export AWS_SECURITY_GROUP_ID="$("${TERRAFORM_BIN}" -chdir=.local/infra/aws output -json | "${GOJQ_BIN}" .security_group_id.value[0])"
+export AWS_SUBNET_ID="$("${TERRAFORM_BIN}" -chdir=.local/infra/aws output -json | "${GOJQ_BIN}" .public_subnets.value[0])"
```

Comment:
If we are only interested in the first value can we instead just output it from terraform

Something like this:
```
output "public_subnets" {
  value = aws_subnet.konvoy_public.[0].id
}

output  "security_group_id" {
  value = aws_security_group.konvoy_ssh.[0].id
}
```

Reply:
This would allow us to remove `gojq`.